elastica/boundary_conditions.py

```diff
@@ -308,8 +308,20 @@ def __init__(self, *fixed_data, **kwargs):
             else:
                 # TODO: This part is prone to error.
                 break
-        self.fixed_positions = np.array(pos)
-        self.fixed_directors = np.array(dir)
+
+        if len(pos) > 0:
+            # transpose from (blocksize, dim) to (dim, blocksize)
+            self.fixed_positions = np.array(pos).transpose((1, 0))
+        else:
+            # initialize empty fixed positions
+            self.fixed_positions = np.zeros((3, 0))
```

Comment:
```suggestion
```
not needed since a non-empty list is always generated in the wrapper.

Reply:
I am not quite sure where the non-empty list is generated in the wrapper, but the test also runs through successfully without this else statement. So I removed it in https://github.com/GazzolaLab/PyElastica/pull/147/commits/c57b88d9cec9e8590f9305365acc6fe59026b022